capstone/src/main/webapp/script.js

```diff
@@ -54,15 +54,39 @@ async function getClubInfo() {
   }
 }
 
+/**Shows or hides the area to post announcements depending on if user is authorized. */
+async function showHidePostAnnouncement () {
+  var params = new URLSearchParams(window.location.search);
+  const query = '/officer?name=' + params.get('name');
+  const response = await fetch(query);
+  const text = await response.text();
+  if (Boolean(text)) {
+    console.log('here');
+    document.getElementById('post-announcement').removeAttribute('hidden');
+  }
+}
+
 /** Accesses and displays club announcement data from servlet. */
 async function loadAnnouncements () {
   var params = new URLSearchParams(window.location.search);
   const query = '/announcements?name=' + params.get('name');
   const response = await fetch(query);
   const json = await response.json();
   const template = document.querySelector('#announcement-element');
+
+  var backgroundColor;
+  const color1 = "#AAA";
+  const color2 = "#BBB";
+  var evenOdd = true;
   for (var announcement in json) {
-    template.content.querySelector('li').innerHTML = json[announcement].content;
+    template.content.querySelector('img').src = 'images/logo.png';
+    template.content.querySelector('#announcement-author').innerHTML = json[announcement].author;
+    template.content.querySelector('#announcement-content').innerHTML = json[announcement].content;
+    template.content.querySelector('#announcement-time').innerHTML = json[announcement].time;
+    backgroundColor = evenOdd ? color1 : color2; //In order to switch background colors every announcement
+    template.content.querySelector('#announcement-container').style = 'background-color:' + backgroundColor;
```

Comment:
I think you should be able to do .style.backgroundColor = backgroundColor;
https://www.w3schools.com/jsref/prop_style_backgroundcolor.asp